packages/paste-core/components/alert-dialog/stories/index.stories.tsx

```diff
@@ -241,7 +241,7 @@ CustomizedAlertDialog.parameters = {
   },
 };
 
-export const CustomizedDestructiveAlertDialog: Story = () => {
+export const CustomizedDestructiveAlertDialog = (): JSX.Element => {
```

Comment:
I think the Story type was recently introduced as part of this PR (https://github.com/twilio-labs/paste/pull/2864/files#diff-b49653ec13744c3f57cc2e680e99cba5d75c5777df5267471f12dfd7a5945f86R221). Any particular reason for reverting these changes?

Reply:
This must have been an outdated commit 🤔